bobanetwork/boba_legacy
# nvmrc version update and validation.

:clipboard: Add associated issues, tickets, docs URL here.

## Overview

update node version `.nvmrc` to 16.16.0 as it's causing issue on gateway to install the latest packages like storybooks and others.

## Changes

Describe your changes and implementation choices. More details make PRs easier to review.

 - Updated .nvmrc version of node to 16.16.0
 - Added node-gyp as dev deps to fix the yarn install prebuild error
 - Spin up the local and executed integrations, unit test to valid the version update
 - Updated the readme with details to build and spin up side services to execute the integration-tests.

## Testing

Execute below steps.
1. prepare build `yarn && yarn clean && yarn build`
2. build side services `bundler aa_deployer`
3. spin up all services ans main boba stack.
3. Execute the integrations tests & unit test should work correctly.
